src/api/Recipes.ts

```diff
@@ -0,0 +1,21 @@
+import axios from "axios";
+
+class RecipesAPI {
+  static getComplexQueryRecipes = async (queryString: string) => {
+    return axios
+      .get("http://localhost:3000/api/recipes/complexSearch", {
```

Comment:
should we hard code the routes or use variables? 
especially if we have multiple environments

Reply:
I agree, will be done by tomorrow